### PR TITLE
HTML/CSS Project #7: Using Bootstrap To Clone The Newsweek Home Page

### PR DESCRIPTION
**Pull request by RailonA**
We build a clone of newsweek.com's homepage using HTML, CSS, and bootstrap. This page works best using the FireFox or Google chrome browser, assets might look out of place using other browsers or resolutions.

The focus of this branch was to create a responsive website, which should have 3 major breaking points: the first one is supposed to be for mobile displays (max-width: 375px); the second one is for tablets displays (max-width: 768px); and the last one is for laptops/monitors (min-width: 768px);

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1080 @@
...
+
+    </main>
+    </div>
+
+
+
+
+
+
+
```
Comment (jubaan) [inline index.html]: - [x] Remove white spaces from HTML file and maintain consistency of spaces, between sections. [OPTIONAL]

File: stylesheets/header.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+
+.nav-header-2{
+  color: #545353;
+  padding: 10px;
+  font-size: .9em;
+}
+
+
+.nav-header-1{
```
Comment (jubaan) [inline stylesheets/header.css]: - [x] Leave only one line between selectors to improve the file and make it look tidy[OPTIONAL]

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1080 @@
...
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Terms &amp; Conditions</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Privacy Policy</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Cookie Policy</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Terms of Sale</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Archive</li>
+                            <li style="font-size: 12px;margin-right: 10px;padding-right: 10px;border-right: 1px solid rgb(237,237,237);margin-bottom: 5px;">Announcements</li>
+                        </ul>
```
Comment (jubaan) [inline index.html]: - [x] these elements, should be links as the original website. the `<ul>` + `<li>` with `<a>` links inside each menu item.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1080 @@
...
+                                                                <li
+                                                                    class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>The Debate</strong></a></li>
+                                                                    <li
+                                                                        class="nav-header-1 nav-item "><a class="nav-link" href="#" ><strong>Vantage</strong></a></li>
+                                                                        <li
+                                                                            class=" nav-header-weather nav-item"><a class="nav-link" href="#" ><strong>Weather</strong></a></li>
+                                                                            <li class="nav-item d-flex"><a class="nav-link" href="#">Search</a><i class="fa fa-search"
+                                                                                    style="color: rgb(99,99,99); margin-top: 10px;"></i></li>
+                                </ul>
+                        </div>
```
Comment (jubaan) [inline index.html]: - [x] Try to maintain the correct indentation through the whole file. [OPTIONAL]

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1080 @@
...
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/all.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+   
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/ionicons/2.0.1/css/ionicons.min.css">
+    <link rel="stylesheet" href="assets/fonts/fontawesome5-overrides.min.css">
+    <link rel="stylesheet" href="stylesheets/normalize.css">
+    <link rel="stylesheet" href="stylesheets/header.css">
+    <link rel="stylesheet" href="stylesheets/main.css">
+    <link rel="stylesheet" href="assets/css/Footer-Clean.css">
+    <link rel="stylesheet" href="assets/css/styles.css">
```
Comment (jubaan) [inline index.html]: - [x] Use paths that start with `./`, notice the dot at the beginning.
```
// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">


// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">

```
  - notice how the correct paths start with a `./` that marks the current position of the file within the project directory. Without that the browser wont know where the path to the asset should start from.

Comment (jubaan) [inline index.html]: COMMENT: check if the project is using all theses elements, and if erase the ones that are not being used.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @ . Thanks for submitting you project!

Great thing this project has and has done well:
:100: Great solid start!
:100: README looks fine

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] Website looks distorted, to solve this create a container or use one that you already have and give it the rule to set a max-width so it will contain elements between the correct width as the original page and margin auto to center that container in the browser independent of the width of the window.
image

- [x] Images look distorted
image

- [x] In smaller screens the menu, goes down a level and distorts the menu
image

- [x] Menu have a white space to the right
image

- [x] Magazine don't look as expected
image

- [x] Elemenest don't look as original
image

- [x] Elements don't look as original
image

- [x] Navbar has a white space at the right on talet view
image

- [x] Navbar doesn't look as expected
image

- [ ] Subscribe and Newsletter section doesn't look like original, they are not centered, the inside elements are not centered, and don't look like the original
image

- [x]  Footer don't look like the original
image

image

- [x] elements should have a space between the image and the text
image

- [x] Top stories should come before featured stories on mobile view
image

- [x] Remove the `.vscode` directory, it is useful for local development but is not necessary for production-ready projects. Only push production-ready files to the repo. Add `.vscode` file to `.gitignore`.

- [x] The linter are not configured. please check [microverse-linters](https://github.com/microverseinc/linters-config/tree/master/html-css) and set up github actions

- [x] The project has duplicated stylesheets, on the stylesheets directory and other in assets css directory. Choose the one that the project will use and change/erase the other or assign it to `.gitinore`

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by mimipeshy** [comment]
### STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hello @RailonA  👋  👋 , good job on working on all the TSE feedback you were given  👏 👏 . However, before we can mark the project as complete let's check on the following

- [x]  On the tablet view, the debate section overlaps the content above it.  ⬇️ 
debate

- [x] The images on the subscribe section on the large view is not similar to the original one. [link](https://www.newsweek.com/)

Original
ori

Clone
subs
 

- [x]  Fix the stylelint and web hint errors in the repo. The stylelint errors can be easily fixed if you run `npx stylelint --fix`. Check on this [link ](https://github.com/microverseinc/linters-config/tree/master/html-css)for configuration.

- [x] The screenshot link in the README file is broken, kindly fix that.

- [x]  The README is missing the `Getting Started` section. This section basically contains instructions on how to use your project, Kindly add that. Check out this [link ](https://github.com/microverseinc/readme-template)for reference.



That's all. 😄 , kindy submit another review when you are done

Regards
Peris Ndanu
Feel free to slack me if you have any questions.

**Comment by RailonA** [comment]
Hello, I have updated all of the requested changes. I have submitted for another code review.

**Review by ioilmio — CHANGES_REQUESTED** [review]
## Hello @RailonA  @ 👋🏼 
### The project is almost ready just fix this image above the footer for the mobile view:
image

##### Your effort to work also during the weekend is admirable but remember to have the right work/life balance during your journey at Microverse, we acre about you and your success;
After this ask for another (and hopefully last) review

*For any further information slack me @illuminatosalvatore*

**Comment by RailonA** [comment]
All requested changes have been changed.

**Comment by mimipeshy** [comment]
### STATUS : APPROVED 🥳  🍾 

Hello @RailonA  👋 👋 , you have done soo well in this project, I have to approve it 👯‍♂️ , congratulations on working this hard! Go ahead and merge. 
Good luck in the next projects ahead 😄 

Regards
Peris Ndanu

**Pull request merged**